### ISSUE TITLE
Add duck logic

### ISSUE THREAD

**Issue by bryandangdev** [bryanjhdang/journeyhacks-2024#6]
Once the popup window for the duck is complete, we should add logic for it.
When you go to a tab that is a part of your blocklist, then:
- The page should freeze / go grey (?)
- The duck should say some words (speech bubble) and be like "are you sewious wight neow"
- You will have to type "I'm sorry I won't do it again"

Additionally, we can maybe add some game logic where the duck progressively gets more pissed off and will eventually just start closing tabs right away as you open them.
